DataManagementSystem/Service/TornadoFileCatalogHandler.py

```diff
@@ -0,0 +1,771 @@
+########################################################################
+# File: FileCatalogHandler.py
+########################################################################
+"""
+:mod: FileCatalogHandler
+
+.. module: FileCatalogHandler
+
+:synopsis: FileCatalogHandler is a simple Replica and Metadata Catalog service
+
+"""
+
+__RCSID__ = "$Id$"
+
+# imports
+import six
+import cStringIO
```

Comment:
Include the Python 3 fallback

Reply:
hum sorry, which Python 3 fallback ? I checked the PR (still not merged...) you made, but it does not mention cStringIO

Reply:
Actually just use `from io import StringIO` and replace `cStringIO.StringIO` with `StringIO`.

`cStringIO` and `StringIO` where replaced with `io.BytesIO` and `io.StringIO` but `io` is available from 2.6 onwards so the new methods can be used.

Reply:
done